app/move/controllers/update/base.test.js

```diff
@@ -625,6 +631,10 @@ describe('Move controllers', function() {
         })
       })
 
+      it('should set the confirmation message', async function() {
```

Comment:
Also, you can remove the `async` from all of these functions as there is no `await` inside them.

Reply:
Zapped